wagtail/images/templates/wagtailimages/images/index_results.html

```diff
@@ -3,24 +3,87 @@
 {% load i18n l10n %}
 
 {% block results %}
-    <div class="nice-padding">
-        <ul class="listing horiz images">
-            {% for image in object_list %}
-                <li>
-                    {% fragment as title_id %}image_{{ image.pk|unlocalize|admin_urlquote }}_title{% endfragment %}
-                    {% include "wagtailadmin/bulk_actions/listing_checkbox_cell.html" with obj_type="image" instance=image aria_describedby=title_id only %}
-                    <a class="image-choice" title="{% if collections %}{{ image.collection.name }} » {% endif %}{{ image.title }}" href="{% url view.edit_url_name image.id %}{% if next %}?next={{ next|urlencode }}{% endif %}">
-                        <figure>
-                            {% include "wagtailimages/images/results_image.html" %}
-                            <figcaption id="{{ title_id }}">
-                                {{ image.title|ellipsistrim:60 }}
-                            </figcaption>
-                        </figure>
-                    </a>
-                </li>
-            {% endfor %}
-        </ul>
-    </div>
+
+    {% if view_mode == "list" %}
+        {% block list_view %}
+            {% panel id="listings" %}
+
+                <table class="listing">
```

Comment:
From Sage: this should be refactored to use Wagtail’s table framework (see document listing), where table columns are defined as Python objects.